commands/cartridge.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/project"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	projectCtx project.ProjectCtx
+	rootCmd    = &cobra.Command{
```

Comment:
Broken spaces between "rootCmd" and "="

Reply:
This is the result of running `go fmt`. I've added a space between definitions and now everything looks fine.